common/main/bm.h

```diff
@@ -150,6 +150,7 @@ extern int Robot_replacements_loaded;
 constexpr std::integral_constant<unsigned, 610> MAX_OBJ_BITMAPS{};
 extern unsigned N_ObjBitmaps;
 extern int extra_bitmap_num;
+extern int Exit_models_loaded;
```

Comment:
This could be a `int8_t`, correct?

Reply:
It could be any type that can contain both a 0 and a 1. In fact, it could even be convertible to a `bool`.